Minio.Examples/Program.cs

```diff
@@ -84,13 +91,28 @@ public static void Main(string[] args)
 
             // WithSSL() enables SSL support in MinIO client
             MinioClient minioClient = null;
-            if (enableHTTPS)
+            if ( port > 0 )
```

Comment:
there is no need for so many client construction methods. If port is not set and https set 443 otherwise 80 if unspecified.

Reply:
Implemented this change.